File: uri/1 - Basics/1962/Main.java
import java.util.*;
class Main{
    public static void main(String []args)
    {
        double t, year;
        double diff;
        Scanner scanner = new Scanner(System.in);
        double n = scanner.nextDouble();
        for(double i=0; i< n; i++){

            t = scanner.nextDouble();

            diff = 2015 - t;
            if (diff <= 0)
                year = diff * -1 + 1;
            else
                year = diff;

            System.out.printf("%.0f %s\n", year, (diff <= 0 ? "A.C." : "D.C."));

        }
    }
}

File: uri/1 - Basics/1962/Main.cpp
#include <iostream>

using namespace std;
int main()
{
    unsigned long long n, t, year;
    long long diff;
    cin >> n;
    for(unsigned long long i=0; i< n; i++){
        cin >> t;

        diff = 2015 - t;
        if (diff <= 0)
            year = diff * -1 + 1;
        else
            year = diff;
        cout << year << " " << (diff <= 0 ? "A.C." : "D.C.") << endl;

    }
    return 0;
}